src/pymax/mixins/socket.py

```
@@ -139,6 +147,188 @@ async def _handshake(self, user_agent: UserAgentPayload) -> dict[str, Any]:
+    async def _recv_data(
+        self, loop: asyncio.AbstractEventLoop, header: bytes, sock: socket.socket
+    ) -> list[dict[str, Any]] | None:
+        packed_len = int.from_bytes(header[6:10], "big", signed=False)
+        payload_length = packed_len & 0xFFFFFF
+        remaining = payload_length
+        payload = bytearray()
+
+        while remaining > 0:
+            min_read = min(remaining, 8192)
+            chunk = await loop.run_in_executor(
+                None, lambda: self._recv_exactly(sock, min_read)
```

Ruff (0.14.8)

Function definition does not bind loop variable `min_read`

(B023)